tests/console/force_cron_run.pm

```diff
@@ -30,18 +30,25 @@ sub settle_load {
 
 sub run {
     select_console 'root-console';
-
     # show dmesg output in console during cron run
     assert_script_run "dmesg -n 7";
 
     # Make sure there's no load before we trigger one via cron.
     settle_load;
     my $before = time;
-    assert_script_run "bash -x /usr/lib/cron/run-crons", 1000;
+    # run cron jobs or systemd timers which can affect system performance and mask systemd timers later
+    assert_script_run('find /etc/cron.{daily,weekly,monthly} -type f -executable -exec echo cron job: {} \; -exec {} \;');
+    assert_script_run(
+        'for i in $(systemctl list-units --type=timer --state=active --no-legend | awk \'{ print $1 }\' | sed \'s/.timer//\'); do
```

Comment:
Because we use sed anyway we can simplify this line to

```perl
for i in $(systemctl list-units --type=timer --state=active --no-legend | sed \'s/.timer.*$//\'); do
```

Reply:
awk \'{ print $1 }\' is required, otherwise it will fail:

http://e13.suse.de/tests/6094#step/force_cron_run/11

Reply:
Unfortunately I can not see the error from outside SUSE ...

You may try `sed -e 's/\(\S\+\)\.timer\s.*/\1/'`

Reply:
yes, I checked your suggestion, it is working, thanks!